styles.css

```diff
@@ -93,3 +84,215 @@ header {
   padding: 5px;
   column-gap: 15px;
 }
+
+.works {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  font-family: 'Poppins', sans-serif;
+  background-color: #e5e5e5;
+}
+
+.project {
+  margin: 10px;
+  margin-bottom: 50px;
+  padding: 10px;
+  border-radius: 4px;
+  text-align: left;
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+}
```

Comment:
- [ ] You did good work but still, your design has some little issues. Kindly adhere to the project requirements as you can see in the attached image down below.

![image](https://user-images.githubusercontent.com/66476329/235255189-1f90f047-2db8-44d5-a0b3-c6251b1fd8cb.png)

- [ ] Kindly take a look again at your design and Figma design. Below the table left-hand side is the Figma design and the right-hand-side is your design. 

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://user-images.githubusercontent.com/66476329/235256152-b64e227c-53b4-4ccc-b005-446e18971830.png)

</td>
<td>

![Screenshot from 2023-04-29 02-30-26](https://user-images.githubusercontent.com/66476329/235256177-2dae0567-15ef-49ee-a851-9379a801f5b0.png)

</td>
</tr>
</table>

Reply:
![Untitled](https://user-images.githubusercontent.com/114990830/235261035-b6341e42-6206-412f-b3fa-a560e307145e.png)
@skyv26 I don't have the button issue in my browser

Reply:
@skyv26

Reply:
@abdozayan12 Kindly look at the other things which I pointed out to improve.